scripts/sct_compute_mtr.py

```diff
@@ -56,52 +56,23 @@ def main(args=None):
 
     fname_mt0 = arguments['-mt0']
     fname_mt1 = arguments['-mt1']
-    remove_temp_files = int(arguments['-r'])
```

Comment:
you also need to remove the entry in `get_parser()`

Reply:
Agreed, fixed in commit [4b41af9](https://github.com/YangHee-Min/spinalcordtoolbox/commit/4b41af9fae9256d27c4ad7ae7d31b11a99291d9b)